pages/dynamic-templates.tsx

```diff
@@ -137,13 +158,16 @@ const DynamicTemplates: NextPageWithLayout<Props> = ({
     decimals: 2,
     enumOptions: initialOptions,
   } as CustomField);
-
+  const [forEmbedCustomField, setForEmbedCustomField] =
+    useState<CustomField | null>(null);
+  const [forEmbedOptions, setForEmbedOptions] = useState(initialOptions);
   const customFieldConfig = {
     key: customField.name?.replace(/\s/, ""),
     type: customField.type,
     label: customField.name,
     description: "Custom field",
     constraints: [{ type: "required" }],
+    forEmbed: forEmbedCustomField ? true : false,
```

Comment:
What does this `forEmbed` stuff do?

Reply:
forEmbed tells it to use the current saved customConfig in the app state